packages/react-components/src/components/ChatMessage/ChatMessageActionMenu.tsx

```diff
@@ -25,41 +24,31 @@ export const chatMessageActionMenuProps = (menuProps: {
   enabled: boolean;
   /** Whether to force showing the action menu button - this has no effect if the action menu button is not enabled */
   forceShow: boolean;
-  menuButtonRef: React.MutableRefObject<HTMLElement | null>;
+  menuButtonRef: React.MutableRefObject<HTMLDivElement | null>;
   onActionButtonClick: () => void;
   theme: Theme;
 }): ChatMessageActionMenuProps | undefined => {
   if (!menuProps.enabled) {
     return undefined;
   }
 
-  const menuClass = mergeStyles(chatActionsCSS, {
-    'ul&': { boxShadow: menuProps.theme.effects.elevation4, backgroundColor: menuProps.theme.palette.white }
-  });
-
   const actionMenuProps: ChatMessageActionMenuProps = {
-    showActionMenu: menuProps.forceShow === true ? true : undefined,
-    iconOnly: true,
-    activeIndex: -1,
-    className: menuClass,
-    onItemClick: () => menuProps.onActionButtonClick(),
-    items: [
-      {
-        children: (
-          <Ref innerRef={menuProps.menuButtonRef}>
-            <Icon
-              iconName="ChatMessageOptions"
-              data-ui-id="chat-composite-message-action-icon"
-              aria-label={menuProps.ariaLabel}
-              styles={iconWrapperStyle(menuProps.theme, menuProps.forceShow)}
-            />
-          </Ref>
-        ),
-
-        key: 'menuButton',
-        indicator: false
-      }
-    ]
+    children: (
+      <div
+        tabIndex={0} //make div focusable, as Icon below is migrated to v9, this can be deleted
```

Comment:
per comment, can this be deleted?

Reply:
We're still using the v8 icons (https://developer.microsoft.com/en-us/fluentui#/controls/web/icon).
v9 Icons (https://react.fluentui.dev/?path=/docs/concepts-developer-icons-icons--page) are from the react-icons package and we've not migrated.